plugins/plugins.go

```diff
@@ -279,10 +286,15 @@ func New(raw []byte, id string, store storage.Store, opts ...func(*Manager)) (*M
 		interQueryBuiltinCacheConfig: interQueryBuiltinCacheConfig,
 	}
 
+	if m.logger == nil {
+		m.logger = sdk.NewStandardLogger()
+	}
+
 	serviceOpts := cfg.ServiceOptions{
 		Raw:        parsedConfig.Services,
 		AuthPlugin: m.AuthPlugin,
 		Keys:       keys,
+		Logger:     m.logger,
```

Comment:
@anderseknert is there a reason we weren't passing this thru before? Was this just an oversight in the original PR?

Reply:
Yeah looks like it was. The logger still gets attached to the config object when instantiating the `Client` [here](https://github.com/open-policy-agent/opa/blob/master/plugins/rest/rest.go#L168), but I believe that was part of an earlier iteration that should have been replaced by injecting it through the `ServiceOptions` rather, which looks like it only got half-done :/ Good to get that fixed! We shouldn't need to keep that previous way of doing it though, and that should be good to remove with this change.